.github/workflows/check_tz_releases.yml

```diff
@@ -25,7 +25,46 @@ jobs:
 
           if [[ $latest_tag_commit != $current_commit ]]; then 
               echo "New tz release available: ${latest_tag}"
+              new_branch="gha-bump-tz-${latest_tag}"
+              git checkout -b $new_branch
+
+              # update the submodule 
+              git -C third_party/tz checkout "${latest_tag}"
+              git add third_party/tz
+
+              # update the url in valhalla_build_timezones 
+              url_old=$(grep -oP "url=\"\Khttps://github.com/.*(?=\")" scripts/valhalla_build_timezones)
+              url_new=$(echo ${url_old} | sed -E "s/download\/[0-9]{4}[a-z]/download\/${latest_tag}/")
+
+              # first make sure the file is actually available
+
+              set +e
+              curl -Is --fail-with-body ${url_new} >/dev/null
+
+              if [[ $? -eq 0 ]]; then 
+                  # only replace if it's available
+                  url_old_escaped=$(echo "${url_old}" | sed 's/[\&/]/\\&/g')
+                  url_new_escaped=$(echo "${url_new}" | sed 's/[\&/]/\\&/g')
+                  sed -i -e "s/${url_old_escaped}/${url_new_escaped}/" scripts/valhalla_build_timezones
+                  git add scripts/valhalla_build_timezones
+
+                  echo "Updated timezone shapefile download URL:"
+                  git diff scripts/valhalla_build_timezones
+              fi
+
+              set -e
+              # commit and push 
+              git commit -m "bumped tz to ${latest_tag}"
+              git push origin 
+
+              # open new PR 
+              gh pr create --base master --head $new_branch --title "Bump tz to ${latest_tag}" --body "Created by workflow run [#${WORKFLOW_RUN_ID}](https://github.com/valhalla/valhalla/actions/runs/$WORKFLOW_RUN_ID/job/${WORKFLOW_JOB_ID})"
```

Comment:
`gh pr create` what is this magic command!? i never knew there was a command line github utility, cool!

Reply:
Should we add the link to the tz commit’s NEWS doc in the PR description so we can conveniently check what changed?